homework45/src/ToDoItem.jsx

```diff
@@ -0,0 +1,29 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { actions } from './store/index.js';
+
+export function ToDoItem({id}) {
+    const todo = useSelector(state => state.todos.find(obj => obj.id === id));
+
+    const dispatch = useDispatch();
+    const deleteToDo = (id) => {
+        dispatch(actions.deleteToDo(id))
+    };
+
+    const toggleToDo = (id) => {
+        dispatch(actions.toggleToDo(id))
+    };
+    
+    return (
+        <li>
+            <label>
+                <input type="checkbox" checked={todo.completed}
+                    onChange={e => toggleToDo(todo.id)}
```

Comment:
you don't need to create additional function and pass an `id`, because you have `todo` in scope

```
const todo = useSelector(state => state.todos.find(obj => obj.id === id));

// ...
const toggleToDo = () => dispatch(actions.toggleToDo(todo.id));

// ...

onChange={toggleToDo}
```

The same for delete todo